en/mulesoft/modules/ROOT/pages/ms_composer_hyperautomation.adoc

```diff
@@ -0,0 +1,31 @@
+= MuleSoft Automation Package 
+
+The https://www.mulesoft.com/automation-pricing-new[MuleSoft Automation Package] enables you to log in to Anypoint Platform to access MuleSoft Composer and xref:rpa-home::index.adoc[MuleSoft Robotic Process Automation (RPA)]. With the Hyperautomation Package, MuleSoft Composer now has everything you need to connect apps and create automations.  
+
+
+With the MuleSoft Automation Package, the following Anypoint Platform features are available to Composer users: 
+
+* xref:access-management::audit-logging.adoc[Anypoint Audit Logging]
+* xref:exchange::index.adoc[Anypoint Exchange] 
+* xref:visualizer::index.adoc[Anypoint Visualizer] 
+* xref:monitoring::index.adoc[Anypoint Monitoring] 
+
+== Access Composer with Hyperautomation Package 
+
+=== Admin: Grant Access to Composer Users 
+
+To access Composer via Anypoint Platform, the Anypoint Admin must first: 
+
+. Create an xref:access-management::managing-your-account.adoc[Anypoint Platform Account] for the users.
+. Grant one of the following xref:ms_composer_prerequisites.adoc#change-user-roles[roles] to each user: 
+.. *Composer Base User*: This user can create and manage their own flows. 
+.. *Composer Super User*: This user can create and manage their own flows. Additionally, this user can take ownership and manage any flows in the Anypoint Composer org. The xref:ms_composer_automation_credits.adoc[Usage Summary] is  only available to those with the Composer Super User role.
+
+=== Access Composer with the Hyperautomation Package  
```

Comment:
Is is really necessary to add "with the Hyperautomation Package" are this instructions different from just the usual way of accessing Composer? 

If that's the Case, I'd suggest a rewording of the titles from this:   

== Access Composer with Hyperautomation Package 
...
=== Admin: Grant Access to Composer Users 
...
=== Access Composer with the Hyperautomation Package  
...

To something like: 


== Access Composer with Hyperautomation Package 
(Add some intro text, like:)
To Access Composer with the Hyperautomation package, first grant access to users, and then log in to Anypoint Platform and open Composer. 
=== Admin: Grant Access to Composer Users 
...
=== Access Composer from Anypoint Platform
...